changelogs/changelog.yaml

```diff
@@ -154,8 +154,9 @@ releases:
       - inventory/terraform_provider - Remove custom `read_config_data()` method (https://github.com/ansible-collections/cloud.terraform/pull/181).
       - inventory/terraform_state - Remove custom `read_config_data()` method (https://github.com/ansible-collections/cloud.terraform/pull/181).
       - inventory/terraform_state - Support for custom Terraform providers (https://github.com/ansible-collections/cloud.terraform/pull/146).
-      release_summary: This release includes bug fixes and new feature for the ``terraform_state``
-        inventory plugin.
+      - modules/terraform - Updated Workspace Logic for TFC/TFE and CLI (https://github.com/ansible-collections/cloud.terraform/pull/194).
```

Comment:
As mentioned in the previous comment. These, per my understanding, are auto-generated by `antsibull-changelog`. Only the fragment should be added.